modules/openy_features/openy_node/modules/openy_node_alert/openy_node_alert.install

```diff
@@ -121,7 +121,39 @@ function openy_node_alert_update_8005() {
     'core.entity_form_display.node.alert.default' => [
       'content.langcode',
       'content.status',
-      'content.addthis',
+    ],
+  ];
+
+  $config_updater = \Drupal::service('openy_upgrade_tool.param_updater');
```

Comment:
THis update won't happen for the guys, who already on 8005 version
Please introduce new hook_update. We must follow the rule - never change old hook_update_N

Reply:
@podarok we introduced a new hook openy_node_alert_update_8006() - GitHub just shown an incorrect diff. The change in openy_node_alert_update_8005 was not obligoritaly to do.